src/outpack/util.py

```diff
@@ -8,7 +8,10 @@
 
 def find_file_descend(filename, path):
     path = Path(path)
-    root = Path(path.root)
+
+    root = path.parent
+    while root != root.parent:
+        root = root.parent
```

Comment:
```suggestion
root = Path(path.anchor)
```

https://docs.python.org/3/library/pathlib.html